ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json

```diff
@@ -81,9 +81,20 @@
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "MANDATORY",
     "PageID": "OrderType",
-    "PageDisplayOrder": 4,
+    "PageDisplayOrder": 3,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "N",
-    "FieldShowCondition": "drawDirectionsOrderSmallClaims = \"No\""
+    "PageShowCondition": "drawDirectionsOrderSmallClaims = \"No\" AND drawDirectionsOrderRequired=\"Yes\""
```

Comment:
Is the second part after the `AND` required, since to my knowledge the `drawDirectionsOrderSmallClaims` field is only reachable when `drawDirectionsOrderRequired` is already `Yes`

Reply:
Nvm, after going through the flow that second condition is needed after all